src/outputs/form_output_prep.py

```diff
@@ -66,19 +53,10 @@ def form_output_prep(
         # outputs_df = pd.concat([outputs_df, ni_full_responses])
         tau_outputs_df = pd.concat([tau_outputs_df, ni_full_responses])
```

Comment:
This line removes the need for a logic gate before hand. 

`tau_outputs_df = pd.concat([tau_outputs_df, ni_full_responses]) if ni_full_responses is not None else tau_outputs_df`

Then we won't need the conditional block and makes the code more readable and computationally efficient.

Again, I know this was not part of your ticket but we may as well generate nice to have code-tidying and efficiency tickets while I am looking at the code.